src/platform_impl/macos/window.rs

```diff
@@ -1151,6 +1151,20 @@ impl WindowExtMacOS for UnownedWindow {
                 .setHasShadow_(if has_shadow { YES } else { NO })
         }
     }
+
+    #[inline]
+    fn focus(&self) {
+        unsafe {
+            self.ns_window.makeKeyWindow();
+        }
+    }
+
+    #[inline]
+    fn focus_and_front(&self) {
+        unsafe {
+            self.ns_window.makeKeyAndOrderFront_(nil);
+        }
+    }
```

Comment:
There's already a way to call this on MacOS, it's done by using `window.set_visible(true)` - so unsure what the benefits are of this?

Reply:
Thanks for your reviews. You'are right, I just found that `window.set_visible(true)` is doing the same thing in this case, the purpose of this api is to focus a specific window and bring it front, may be used in multiple window situations. The`set_visible` function name just like to set the window's visibility as it does on windows platform.